js/attendeeContact.js

```diff
@@ -28,17 +28,18 @@ function getAttendeeInfo() {
     //var badgeNumber = "1234";     //For testing
 
     var attendeeName = "".concat(document.getElementsByName('attendee.firstName')[0].value," ",document.getElementsByName('attendee.lastName')[0].value);
-    var attendeeBadgeName = document.getElementsByName('attendee.customDataList[4].value')[0].value;
+    var attendeeBadgeName = $("label:contains('Badge Name'):first").next().find("input").val();
 
     if (attendeeBadgeName == "" || attendeeBadgeName == null) {
         attendeeBadgeName = attendeeName;
     }
 
-    var numberActiveBadges = parseInt(document.getElementsByName('attendee.customDataList[7].value')[0].value);
+    var numActBadgesFromElement = $("label:contains('Number of Active Badges'):first").next().find("input").val();
+    var numberActiveBadges = numActBadgesFromElement === "" ? 0 : parseInt($("label:contains('Number of Active Badges'):first").next().find("input").val());

```

Comment:
why not: 
var numberActiveBadges = numActBadgesFromElement === "" ? 0 : parseInt(numActBadgesFromElement);